src/plugins/guided_onboarding/public/constants/guides_config/security.ts

```diff
@@ -52,11 +72,16 @@ export const securityConfig: GuideConfig = {
     },
     {
       id: 'alertsCases',
-      title: 'Alerts and cases',
+      title: i18n.translate('guidedOnboarding.securityGuide.alertsStep.title', {
+        defaultMessage: 'Alerts and cases',
```

Comment:
```suggestion
        defaultMessage: 'Manage alerts and cases',
```

Reading this again, it needs a verb to be parallel with the other steps.